### PR TITLE
Done

### PR DESCRIPTION
**Pull request by wig-wig**

### CODE REVIEW

File: lib/arrays.md [hunk 1]
```diff
@@ -5,44 +5,78 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
...
 
 # The previous question asks about finding, for example, the 6th element in an Array. Is it possible to find the **-6th** (Notice the negative symbol!) element in an Array? What does that even mean?
 
-Replace me with your response...
+Yes. It means the - is calling to the last element of an array as -1 so it will return the 6th element from the end.
 
 # How would you perform an operation on every element inside an Array?
 
-Replace me with your response...
+By using the each and map methods.
```
Comment (sumeetjain) [inline lib/arrays.md:39]: What is the difference between the `each` and `map` methods? Or are they the same?

File: lib/arrays.md [hunk 2]
```diff
@@ -5,44 +5,78 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
...
 
 # How would you reverse the order of elements in an Array?
+Using .reverse
+
+arr = [1, 2, 3]
+arr.reverse
+
+# How would you convert a String `"Sumeet Jain"` into an Array `["Sumeet", "Jain"]`? 
+
+"Summet Jain".split
```
Comment (sumeetjain) [inline lib/arrays.md:80]: I think you missed this question: How about to `["S", "u", "m", "e", "e", "t", " ", "J", "a", "i", "n"]`?

File: lib/arrays.md [hunk 3]
```diff
@@ -5,44 +5,78 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
...
+You can use the push or << method. 
+
+arr = ["yes", "no"] 
+arr.push("maybe") or arr << "maybe"
 
 # Given the Array `["Laura", "Fiona", "Tori"]`, how would you replace `"Fiona"` with `"Florence"` so that you end up with `["Laura", "Florence", "Tori"]`?
 
-Replace me with your response...
+arr = ["Laura", "Fiona", "Tori"]
+arr[1].replace("Florence")
```
Comment (sumeetjain) [inline lib/arrays.md:56]: It's true that `arr[1].replace("Florence")` does replace `"Fiona"` with `"Florence"`, and I'll always encourage y'all to find and appreciate the various ways to achieve a goal. So nice job :)

I'd also like to point out that `arr[1] = "Florence"` is the more common way to do this. Just an FYI–no reply or revision on your part necessary.

Comment (wig-wig) [inline lib/arrays.md:56]: Ah ha. I see. That's cleaner. :+1:

File: lib/arrays.md (hunk 2 above)
Comment (wig-wig) [inline lib/arrays.md:80]: Weird... If you're speaking about the last question in the .md, I answered it. Not sure why it didn't show. 

Let me know if didn't come through this time.

Comment (sumeetjain) [inline lib/arrays.md:80]: Hm, I'm still not seeing it. Perhaps just add the code in your reply to this comment?

Comment (wig-wig) [inline lib/arrays.md:80]: Oops!!! I did miss that portion of the question. Sorry. Eyes were playing tricks on me. 

The code would be -
"Sumeet Jain".split ("")

**Comment by sumeetjain** [comment]
:+1:

**Pull request closed without merging**